cloudinit/util.py

```diff
@@ -1268,6 +1273,11 @@ def find_devs_with_netbsd(criteria=None, oformat='device',
     return []
 
 
+def find_devs_with_openbsd(criteria=None, oformat='device',
+                           tag=None, no_cache=False, path=None):
+    return ["/dev/cd0a"]
```

Comment:
Should there at least be an existence check for this instead of returning it unconditionally? (If not, then an explanatory comment in here would be good, I think.)

Reply:
Yeah, actually I should reuse the logic from cloudinit/sources/DataSourceNoCloud.py and add some test coverage. I will do that later.

Reply:
done